src/Maths/Silk.NET.Maths/Scalar.Exp.cs

```diff
@@ -0,0 +1,244 @@
+﻿// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+/*
```

Comment:
Pleaseadd an accompanying comment to the second license header.

Medium priority.

Reply:
Not sure what you mean?

Reply:
Got it.